src/main/scala/mastermind/Mastermind.scala

```diff
@@ -13,19 +13,36 @@ val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
 def getRandomColor(): Color =
-  ???
+  Random.shuffle(validColors).head
```

Comment:
Huge fan of the way you did this, it is much more simple and easy to read compared to my solution